app/lib/admin/models.dart

```diff
@@ -286,6 +305,27 @@ class ModerationActionLog {
       _$ModerationActionLogFromJson(json);
 
   Map<String, Object?> toJson() => _$ModerationActionLogToJson(this);
+
+  /// Returns the list of subjects that have a final state as moderated.
+  /// If a package version and also a package is moderated, returns
+  /// only the package.
+  List<String> moderatedSubjects() {
```

Comment:
Find state is difficult. Let's not try to do that.
Let's just return the log and let humans read it?